### DISCUSSION TITLE
Component Discovery: Created Components aren´t visible

### DISCUSSION THREAD

**Discussion by Zanragul** [WeblateOrg/weblate#7673] [Q&A]
Hi,
I´m trying to localize (german -> english) a big project via weblate. We have many .resx files that need translations so we thought it would be a good idea to use the Component Discovery addon. I first made a Component called "Glossar" so that I can install the addon. My addon configurations look like this: 
![image](https://user-images.githubusercontent.com/28594870/169485089-55c6ddc1-a23e-4297-88c7-df5e2eb88e4b.png)
All seems to be working so far. I get the correct files displayed and when I save it again the log looks like it is creating the components I want:
`Creating component UBS.WinForms.Controls.Html.UBSHtmlEditor
Creating component UBS.WinForms.Debugging.frmBusinessobjectStateSnapshotViewer
Creating component UBS.WinForms.Wizard.UBSSelectionWizard
Creating component UBS.WinForms.Wizard.Base.BaseWizardForm
Creating component UBS.WinForms.Wizard.Base.UBSWizardForm
Creating component UBS.WinForms.Printing.frmPrintingProgress
Creating component UBS.WinForms.UserDefiniedQueryUI.UBSSelectionCriteriaEditControl
Creating component UBS.WinForms.UserDefiniedQueryUI.UBSSelectionCriteriaEditControlForBool
Creating component UBS.WinForms.UserDefiniedQueryUI.UBSUserDefiniedQueryBuilderControl
Creating component UBS.WinForms.UserDefiniedQueryUI.UBSUserDefiniedQuery`

But I cant see the new components in my project. There is still only my Glossar component from the beginning.
What am I missing?

Thanks in advance

**Comment by nijel** [comment]
Maybe Celery is not running? The components will be created in a Celery task.
